Server/jail.local

```diff
@@ -241,12 +241,12 @@ maxretry = 2
 
 # http://linuxaria.com/howto/how-to-protect-apache-with-fail2ban
 
-[apache-phpmyadmin]
-enabled  = true
-port     = http,https
-filter   = apache-phpmyadmin
-logpath  = /var/log/apache*/*error.log
-maxretry = 3
+#[apache-phpmyadmin]
```

Comment:
Why comment these out?

Reply:
you are correct - i dont need to comment out - i may have gotten a bit over zealous.  it just assumes that the user is using phpmyadmin which they may not (not part of the bluesky setup or instructions).  i can uncomment in this PR and get this jail back to what it was.